packages/fast-components-react-msft/src/select/select.tsx

```diff
@@ -26,6 +27,7 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, {}> {
                 {...this.unhandledProps()}
                 managedClasses={this.props.managedClasses}
                 disabled={this.props.disabled}
+                menu={this.props.menu}
```

Comment:
This doesn't quite resolve 2250, but it lets you fix it in your code. We should address it as the default stying though. Can you wrap the default menu in a Background and add neutralLayerFloating for its color?